common/src/tap/receipt_store.rs

```diff
@@ -21,38 +104,53 @@ impl ReceiptStore for IndexerTapContext {
         &self,
         receipt: ReceiptWithState<Checking>,
     ) -> Result<u64, Self::AdapterError> {
+        let db_receipt = DatabaseReceipt::from_receipt(receipt, &self.domain_separator)?;
+        self.receipt_producer.send(db_receipt).await.map_err(|e| {
+            error!("Failed to queue receipt for storage: {}", e);
+            anyhow!(e)
+        })?;
+
+        // We don't need receipt_ids
```

Comment:
The issue is that all errors within that task are unhandled and this isn't visible to the reader now.

Reply:
Same thing here, ask what the indexers would prefer.
All the business is going through the indexer-service, so whatever is does, it will be very impactful to the indexers' business.